mathics/builtin/numbers/calculus.py

```diff
@@ -1449,9 +1451,7 @@ def apply(self, f, x, x0, evaluation, options):
             method = method.value
 
         # Determine the "jacobian"
-        if method in ("Newton",) and options["System`Jacobian"].sameQ(
-            Symbol("Automatic")
-        ):
+        if method in ("Newton",) and options["System`Jacobian"].sameQ(SymbolAutomatic):
```

Comment:
`in ("Newton",)` -> `== "Newton"`

Reply:
Yes, all this needs more work. Y just tried to have a working branch to see the differences and try to improve it.